doc/libf3d/OPTIONS.md

```diff
@@ -17,7 +17,9 @@ See the exhaustive list below, but note that this may change in the future.
 
 Option|Type<br>Default<br>Trigger|Description|F3D option
 :---:|:---:|:---|:---:
-scene.animation.index|int<br>0<br>load|Select the animation to load.<br>Any negative value means all animations.<br>The default scene always has at most one animation.|\-\-animation-index
+scene.animation.index|int<br>0<br>load|Select the animation to load.<br>Any negative value means all animations (glTF only).<br>The default scene always has at most one animation.|\-\-animation-index
```

Comment:
If multiple values are supported why not actually have this argument accept multiple values rather that rely on a magic value?

Reply:
Two reasons:
1. Only gltf supports multiples animations, not other importers sadly
2. Even if we added support of multiples arguments, being able to show them all in a simple command is practical as this is a standard usecase for many users.

Reply:
I think what @snoyer is suggesting could be useful if there are many animations and you want to enable only a subset.  
Out of the scope of this PR though but we could think about it.

Reply:
first fixing #841  is needed imo.